### DISCUSSION TITLE
Okay, who is still using this code?

### DISCUSSION THREAD

**Discussion by suaefar** [suaefar/ryzen-test#35] [Q&A]
Are there still faulty first generation Ryzen CPUs out there?

Did the code help you to detect them?

**Comment by aureliegrebe** [comment]
I haven't used your code yet, but I've been having copious amounts of segfaults while attempting to compile tensorflow and then gcc-8 on a ryzen 3200g. Has anyone else hear of this problem on non-first gen ryzen?

**Comment by suaefar** [comment]
Okay, if you do, try to use an Ubuntu 17.04. live system. (Might be problematic with the 3200G?)

I also have a Ryzen 3 3200G in a Deskmini A300, which did not give problems until now.
There were people reporting segfault problems with 2gen Ryzen, but I do not remember a case where it finally was clearly due to this bug.
My experience with the Ryzen platform is that there are several ways to achieve segfaults.
The most common maybe is unstable/overclocked RAM.

**Reply by aureliegrebe** [reply]
Well, I turned my RAM speed down to 3200 (the native speed is 3600) and it appears to have solved the issue!

**Reply by suaefar** [reply]
I think for that CPU max supported officially is 3000, and 3600 was probably too high.
If you depend on a reliable system, you can check with these two tools:
1) Memory test: https://www.memtest86.com/
2) CPU stress test: https://www.mersenne.org/download/
